### PR TITLE
Feat: Modify achievement window

### LINKED ISSUE #19: Modify-Achievement-Window
## Problem Description

Modify the size and position of the achievement window so that it is not disturbed by the dividing line.

## Todo List

-   [ ] Modify the drawAchievement method of the DrawManagerImpl class.

## Feature Description
The position and size of the achievement window will be modified.

## Additional Information

The achievement window should not interfere with the dividing line and the player.

### PR DESCRIPTION
**Pull request by min5421**
## What
This PR modify achievement window. It includes the following changes:
- Modify the position of the achievement window downward
- Modify the line spacing of text in the achievement window to be narrow
- Modify the text of the achievement window to always be centered

## Why
Not to interfere with the dividing line and the player.

## Related Issue
Closes #19

## How to Test
1.  Run the core class
2. Check if the achievement window doesn't interfere the dividing line and the player when you shoot
3. Check if the achievement text is in the center of the achievement window
4. Check if it works if you change it to another string instead of "sentence"

## Screenshots
achievement2

## Additional Information
I didn't increase the size because there was space, so I modify the position of the achievement window downward
Achievement text changed from left to center alignment.

### CODE REVIEW

**Review by yunsful — APPROVED** [review]
스크린샷 2024-10-06 오후 6 24 36
스크린샷 2024-10-06 오후 6 25 52

- Confirmed that the achievement window appears when a bullet is fired.
- Verified whether the change in the sentence is reflected.

**Review by kojesung — CHANGES_REQUESTED** [review]
The functionality is well implemented, but it would be beneficial to include author information and comments explaining the changes made.
image

image

**Review by LeeHyunWoo02 — APPROVED** [review]
confirmed that the achievement screen is displayed on the game screen safely.

image



and confirmed that it works without any problems even after modifying the output sentence.

image

**Comment by aeioiie** [comment]
LGTM

File: src/HUDTeam/DrawManagerImpl.java [hunk 1]
```diff
@@ -83,6 +85,7 @@ public static void drawAchievement(final Screen screen, String achievementText)
         backBufferGraphics.setColor(Color.white);
         backBufferGraphics.drawRect(screen.getWidth() - width - 3, screen.getHeight() - height - 18, width, height);
 
+        //Modify the location of the window and the text - Jo Minseo/HUD team
```
Comment (kojesung) [inline src/HUDTeam/DrawManagerImpl.java:88]: I have confirmed that additional comments have been added.

**Review by kojesung — APPROVED** [review]

**Review by aeioiie — COMMENTED** [review]
LGTM

**Review by aeioiie — APPROVED** [review]

**Comment by sml-logs** [comment]
ScreenShot 2024-10-06 오후 6 46 34
Confirmed that the achievement text is displayed properly on the screen. 
ScreenShot 2024-10-06 오후 6 55 03
Confirmed that it works well even if the text is changed.

**Pull request merged**